README.md

```diff
@@ -12,6 +12,7 @@
   * MongoDB
   * MySQL
   * gRPC
+  * Memcached
```

Comment:
What is this about with the upgrade?

Reply:
Just added missing check description that was merged in https://github.com/hellofresh/health-go/pull/54